### DISCUSSION TITLE
[QUESTION] Does Vercel Blob support metadata editing?

### DISCUSSION THREAD

**Discussion by CalvinDudd** [vercel/community#5591] [Feature Question] [answered]
### Question

I am apart of a student organization at a University, we are using Vercel blob to store mp3/mpeg files for our web app. 
We wish to quickly upload / download to and from the blob.
To do this we believe editing the metadata of MP3/MPEGs in the blob is necessary. We however learned that his wasn't possible using Blob. Speed is of the upmost importance for our app, is there any work around?

I want to confirm that my current understanding of Vercel Blob is as I described, otherwise we may have to switch away from Blob.
Let me know, and thanks in advance!

**Comment by amyegan** [comment, marked as answer]
Vercel Blob object metadata can be read after uploading. You can learn more about current capabilities in the docs:
- [Vercel Blob](https://vercel.com/docs/storage/vercel-blob#)
  - [Blob SDK Reference](https://vercel.com/docs/storage/vercel-blob/using-blob-sdk)
  - [Server Uploads with Vercel Blob](https://vercel.com/docs/storage/vercel-blob/server-upload#)
  - [Client Uploads with Vercel Blob](https://vercel.com/docs/storage/vercel-blob/client-upload#)

**Reply by CalvinDudd** [reply]
Thanks!
